Sources/SimplyCoreAudio/Public/SimplyCoreAudio+Aggregate.swift

```diff
@@ -51,6 +51,15 @@ public extension SimplyCoreAudio {
 
         return AudioDevice.lookup(by: deviceID)
     }
+    
+    @available(*, deprecated, message: "mainDevice: is preferred spelling for first argument")
```

Comment:
not sure if there is a way to achieve auto-fixable warning for this (as with `renamed:` for signatures where appears in name vs args)